### PR TITLE
non agg query execution

### PR DESCRIPTION
**Pull request by shz117**

### CODE REVIEW

File: datanode/client/query_client.go [hunk 1]
```diff
@@ -46,7 +46,33 @@ type aqlRespBody struct {
...
+		err = errors.New(fmt.Sprintf("invalid response from datanode, resp: %s", bs))
+		return
+	}
+	result = respBody.Results[0]
+	utils.GetLogger().With("host", host, "query", query, "result", result).Debug("datanode query client Query succeeded")
+	return
+}
+
+func (dc *dataNodeQueryClientImpl) QueryRaw(ctx context.Context, host topology.Host, query queryCom.AQLQuery) (bs []byte, err error) {
+		bs, err = dc.queryRaw(ctx, host, query)
```
Comment (voyager-dw) [inline datanode/client/query_client.go]: lint?

File: broker/query_plan_agg.go [hunk 2]
```diff
@@ -140,7 +140,7 @@ type ScanNode struct {
 	dataNodeClient dataCli.DataNodeQueryClient
 }
 
-func (sn *ScanNode) Execute(ctx context.Context) (result queryCom.AQLQueryResult, err error) {
+func (sn *BlockingScanNode) Execute(ctx context.Context) (result queryCom.AQLQueryResult, err error) {
 	trial := 0
 	for trial < rpcRetries {
```
Comment (voyager-dw) [inline broker/query_plan_agg.go]: where is the rpcRetries definition?

File: datanode/client/query_client.go (hunk 1 above)
Comment (shz117) [reply datanode/client/query_client.go]: already updated

File: broker/query_plan_agg.go (hunk 2 above)
Comment (shz117) [reply broker/query_plan_agg.go]: it's a const on the top of the file

**Pull request merged**